packages/next-intl/package.json

```diff
@@ -120,27 +120,27 @@
   "size-limit": [
     {
       "path": "dist/production/index.react-client.js",
-      "limit": "15.785 KB"
+      "limit": "16.055 KB"
```

Comment:
I've added a note to https://github.com/amannn/next-intl/issues/779 to make sure we target modern browsers, currently we're compiling too much syntax (e.g. rest params).